source/adapter.screenshare.js

```diff
@@ -0,0 +1,161 @@
+(function () {
+
+  'use strict';
+
+  var tempGetUserMedia = null;
+
+  if (window.navigator.mozGetUserMedia) {
+    tempGetUserMedia = window.navigator.getUserMedia;
+
+    window.navigator.getUserMedia = function (constraints, successCb, failureCb) {
+      constraints = constraints || {};
+
+      if (constraints.video ? !!constraints.video.mediaSource : false) {
```

Comment:
Also what is constraints.video.mediaSource == 'camera' ...
I feel like we should define what we expect from the user when they want to get a screensharing stream.
constraints.video.mediaSource == 'screen'